src/app/clusters/fabric/fabric.cpp

```diff
@@ -20,10 +20,13 @@
  * @brief Implementation for the Descriptor Server Cluster
  ***************************************************************************/
 
+#include "CHIPDeviceManager.h"
```

Comment:
What is this include for ? `CHIPDeviceManager` is something that is part of `examples`.  We should not rely on it.